x-pack/plugins/ingest_pipelines/public/application/lib/utils.ts

```diff
@@ -5,21 +5,25 @@
  * 2.0.
  */
 
-export const stringifyJson = (json: any): string =>
-  Array.isArray(json) ? JSON.stringify(json, null, 2) : '[\n\n]';
+export const stringifyJson = (json: any, isList: boolean = true): string =>
```

Comment:
[nit] I might be a little more explicit here. What do you think about `isArray` or `renderAsArray` instead of `isList`?

Reply:
Yes, indeed `renderAsArray` sounds like much more meaningful parameter name so I renamed it to this. Thanks for the suggestion!